src/create/aws/create-table-index/index.js

```diff
@@ -13,11 +13,11 @@ module.exports = function _createIndex(params, callback) {
   var attr = params.index[name]
 
   parallel([
-    function _createStaging(callback) {
-      index(`${params.app}-staging-${name}`, attr, callback)
+    function _createStaging(cb) {
+      index(`${params.app}-staging-${name}`, attr, cb)
     },
-    function _createProduction(callback) {
-      index(`${params.app}-production-${name}`, attr, callback)
+    function _createProduction(cb) {
+      index(`${params.app}-production-${name}`, attr, cb)
```

Comment:
This was causing me issues running `npm run create` locally, resulting in this error

```
.../arc-workflows/node_modules/aws-sdk/lib/request.js:31
            throw err;
            ^

TypeError: callback is not a function
    at Response._tbl (.../arc-workflows/src/create/aws/create-table-index/_create-index.js:36:15)
    at Request.<anonymous> (.../arc-workflows/node_modules/aws-sdk/lib/request.js:364:18)
    at Request.callListeners (.../arc-workflows/node_modules/aws-sdk/lib/sequential_executor.js:105:20)
    at Request.emit (.../arc-workflows/node_modules/aws-sdk/lib/sequential_executor.js:77:10)
    at Request.emit (.../arc-workflows/node_modules/aws-sdk/lib/request.js:683:14)
    at Request.transition (.../arc-workflows/node_modules/aws-sdk/lib/request.js:22:10)
    at AcceptorStateMachine.runTo (.../arc-workflows/node_modules/aws-sdk/lib/state_machine.js:14:12)
    at .../arc-workflows/node_modules/aws-sdk/lib/state_machine.js:26:10
    at Request.<anonymous> (.../arc-workflows/node_modules/aws-sdk/lib/request.js:38:9)
    at Request.<anonymous> (.../arc-workflows/node_modules/aws-sdk/lib/request.js:685:12)
```